### PR TITLE
Removed axios, to fetch API, some new changes

### PR DESCRIPTION
**Pull request by kysnsoft**

### CODE REVIEW

File: src/pages/DotaAPI/FetchFunction.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const getAPI = async arg => {
+    let response = await fetch(arg);
```
Comment (yuhosns) [inline src/pages/DotaAPI/FetchFunction.jsx]: will merge, but i would say https://api.opendota.com/api should be include here

Comment (kysnsoft) [reply src/pages/DotaAPI/FetchFunction.jsx]: noted

Comment (kysnsoft) [reply src/pages/DotaAPI/FetchFunction.jsx]: In this case, will be 

let response = await fetch(`https://api.opendota.com/api${arg}`);

?

Comment (yuhosns) [reply src/pages/DotaAPI/FetchFunction.jsx]:
```
const API_HOST = https://api.opendota.com/api

const callApi = async (endpoint){
  let response = await fetch(`API_HOST/${endpoint}`);
}
```

**Pull request merged**